enhancements/ingress/optional-ingress-hypershift.md

```diff
@@ -0,0 +1,323 @@
+---
+title: optional-ingress-hypershift
+authors:
+  - "@alebedev87"
+reviewers:
+  - "@Miciah"
+  - "@bparees"
+  - "@cewong"
+  - "@wking"
+approvers:
+  - "@Miciah"
+  - "@bparees"
+  - "@wking"
+api-approvers:
+  - N/A
+creation-date: 2023-06-02
+last-updated: 2023-06-02
+tracking-link:
+  - https://issues.redhat.com/browse/NE-1129
+see-also:
+  - "/enhancements/installer/component-selection.md"
+  - "/enhancements/ingress/aws-load-balancer-operator.md"
+replaces:
+superseded-by:
+---
+
+# Make Ingress Operator optional on HyperShift
+
+## Summary
+
+This enhancement describes the implementation of a new capability to enable or disable the default ingress of the OpenShift cluster.
+The new capability (from here on called the ingress capability) inhibits installation of [the cluster ingress operator](https://github.com/openshift/cluster-ingress-operator) and [its custom resources](#ingress-operator-apis).
+This enhancement focuses on the requirements for [RFE-3395](https://issues.redhat.com/browse/RFE-3395) which targets the ROSA clusters spawned and managed by HyperShift. The implementation of the ingress capability on the standalone OpenShift is out of the scope of this enhancement.
+
+## Motivation
+
+Users of the ROSA managed OpenShift service would like to use [AWS load balancers](/enhancements/ingress/aws-load-balancer-operator.md) as a way to accept the ingress traffic into the cluster.
+The cluster ingress operator becomes unnecessary and can be disabled to:
+- reduce the resource consumption on
+  - the HyperShift user clusters (ingress controllers)
+  - the HyperShift hosted control plane (operator)
+- make the HyperShift hosted control plane lighter and simpler, therefore more manageable
+- enable the HyperShift cluster users to make their choice for the cluster ingress
+
+### Goals
+- Implement the new ingress capability.
+- Add the ingress capability to the ingress operator's payload.
+- Make the dependent cluster operators tolerate the absence of the ingress operator.
+- Do not lock the subsequent implementation for the standalone OpenShift.
```

Comment:
Can you clarify that the goal is to implement the functionality required _for HyperShift only_, but that we want to leave a path for implementing the functionality for standalone too?
```suggestion
- Implement the new ingress capability for use by HyperShift.
- Add the ingress capability to the ingress operator's payload.
- Make the dependent cluster operators tolerate the absence of the ingress operator.
- Avoid adding impediments to implementing the ingress capability for standalone OpenShift.
```